code/lib/preview-web/src/Preview.tsx

```diff
@@ -41,7 +41,10 @@ const STORY_INDEX_PATH = './index.json';
 
 export type MaybePromise<T> = Promise<T> | T;
 
-export class Preview<TFramework extends AnyFramework> {
+const renderToDOMDeprecated = deprecate(() => {},
+`\`renderToDOM\` is deprecated, please rename to \`renderToRoot\``);
```

Comment:
If there's no implementation then simply import `deprecate` from client-logger instead. It just prints the warning once.